gumgum/gumdrops
# 108 upgrade react to 18

this looks like a lot but don't worry!

### Changes
-   Upgrade Storybook version to 7 * This required an update to all the stories. They have been moved from `_stories` folder to `stories` folder, so ignore all the changes in `_stories` as they are just deletes.
-   Upgrade React to 18
-   Added color swatches page
-   Deprecate CircularThumbnail, NumberCircle, ProgressBar, Trend, Avatar, Breadcrumbs, BreadcrumbsWrapper *Not used by any of our projects.
-   Update examples in documentation to use hooks *I've tested these all in codesandbox but you can check them out too
-   Update gumdrops docs
-   Update tests to React Test Library/react-test-renderer since Enzyme is deprecated

### To Test
- Go to the build link and ensure each of the components works as expected https://feature-108-upgrade-react--gumdrops.netlify.app/
